pom.xml

```diff
@@ -122,19 +107,14 @@
         <dependency>
             <groupId>com.sun.xml.messaging.saaj</groupId>
```

Comment:
Are the libraries related to Web Services still required? If you don't use the old SOAP interface, I think we can remove them.

Reply:
Good point. We do not need this and we decrease the size of a resulting WAR file from 71.7MB to 48.8MB by removing these dependencies. 👍